provider/app.py

```diff
@@ -157,6 +157,14 @@ def main():
     streamHandler.setFormatter(formatter)
     logger.addHandler(streamHandler)
 
+    # also log to file if /logs is present
+    if os.path.isdir('/logs'):
+        fh = logging.FileHandler('/logs/kafkatriggers.log')
+        formatter = logging.Formatter(
+            '%(asctime)s [%(levelname)s]: %(message)s')
+        fh.setFormatter(formatter)
+        logger.addHandler(fh)
+
     restoreTriggers()
 
     port = int(os.getenv('PORT', 5000))
```

Comment:
What are the advantages of having the port as an environment variable vs a Docker run argument?